src/main/java/Task.java

```diff
@@ -0,0 +1,48 @@
+/**
+ * This class represents a task that can be created
```

Comment:
Do you think this comment is misleading? Since Task cannot be created but rather Task's children classes.

Reply:
I agree with @jlxw48